code/modules/clothing/suits/armor.dm

```diff
@@ -207,39 +209,47 @@
 			return
 		H.forceMove(picked)
 		H.rad_act(rad_amount)
-		reactivearmor_cooldown = world.time + 100
+		reactivearmor_cooldown = world.time + reactivearmor_cooldown_duration
 		return 1
 	return 0
 
 /obj/item/clothing/suit/armor/reactive/teleport/emp_act(severity)
 	..()
-	reactivearmor_cooldown = world.time + 200
+	reactivearmor_cooldown_duration = world.time + 200
```

Comment:
you're aware this will make the cooldown pretty much infinite the next time the armor is activated?